project/templates/configs/springboard.ini

```diff
@@ -0,0 +1,93 @@
+[app:springboardfrontend]
+use = egg:springboard-{{ app_type }}
```

Comment:
This should use `_` instead of `-` like https://github.com/universalcore/unicore-mc/pull/78/files#diff-5e7b9f95bfe899b8e57feed82a33ca49R38